versions.json

```diff
@@ -5,6 +5,12 @@
             "npmName": "@vaadin/bundles"
         }
     },
+    "react": {
+        "react-components": {
+            "jsVersion": "1.3.0-alpha6",
+            "npmName": "@hilla/react-components"
+        }
+    },
```

Comment:
Is adding this section safe?

Reply:
It is fine to add it here. This file will not be deployed anywhere.